site/docs/v1/tech/apis/system.adoc

```diff
@@ -39,7 +40,7 @@ This API is used to retrieve the System Configuration.
 === Request
 
 [.api-authentication]
-link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas]] Retrieve the System Configuration
+link:/docs/v1/tech/apis/authentication#api-key-authentication/[icon:lock[type=fas]] Retrieve the System Configuration
```

Comment:
Did you mean to add a `/` to the end of this URL? In general, I think most pages will end with a slash, but this last URL segment is a URI fragment since it follows a `#` symbol, and is not part of base URI.  I think adding the trailing `/` will break the deep link on the page.  

Same goes for the rest of the the links to `../tech/apis/authentication`.

Reply:
Oh, can revert.  I thought that these were just missing the `/` to be fully qualified links.  Didn't realize the bit about a url fragment and the backslash.